webapp-django/crashstats/crashstats/utils.py

```diff
@@ -318,7 +320,7 @@ def __init__(self, f, dialect=csv.excel, encoding="utf-8", **kwds):
         self.encoder = codecs.getincrementalencoder(encoding)()
 
     def writerow(self, row):
-        self.writer.writerow([unicode(s).encode("utf-8") for s in row])
+        self.writer.writerow([str(s).encode("utf-8") for s in row])
```

Comment:
This doesn't work in Python 2. It needs to be a unicode--not a str. You can use `six.text_type` here and that'll work.

Reply:
This is true of most of the changes from `unicode` to `str` in this PR. I'll leave off commenting on the others.